--- infinispan-8.2.4.Final/lucene/lucene-directory/src/main/java/org/infinispan/lucene/impl/DirectoryLucene.java.orig	2020-03-14 20:37:18.659307745 -0400
+++ infinispan-8.2.4.Final/lucene/lucene-directory/src/main/java/org/infinispan/lucene/impl/DirectoryLucene.java	2020-03-14 20:36:01.254296110 -0400
@@ -64,6 +64,18 @@ class DirectoryLucene extends Directory
     * {@inheritDoc}
     */
    @Override
+   public void rename(String source,String dest) {}
+
+   /**
+    * {@inheritDoc}
+    */
+   @Override
+   public void syncMetaData() {}
+
+   /**
+    * {@inheritDoc}
+    */
+   @Override
    public void deleteFile(final String name) {
       ensureOpen();
       deleteExecutor.execute(new DeleteTask(name));
