From: Build-team<buildteam@openeuler.org>
Implement of abstract function extended from class Directory
--- lucene/lucene-directory/src/main/java/org/infinispan/lucene/impl/DirectoryLucene.java.orig	2016-08-11 15:23:06.000000000 +0000
+++ lucene/lucene-directory/src/main/java/org/infinispan/lucene/impl/DirectoryLucene.java	2020-02-29 08:28:51.774738108 +0000
@@ -59,6 +59,18 @@ class DirectoryLucene extends Directory
       this.indexName = indexName;
       this.lockFactory = lf;
    }
+   
+   /**
+    * {@inheritDoc}
+    */
+   @Override
+   public void rename(String source,String dest) {}
+
+   /**
+    * {@inheritDoc}
+    */
+   @Override
+   public void syncMetaData() {}
 
    /**
     * {@inheritDoc}
